src/lib/tooltip/tooltip.ts

```diff
@@ -223,12 +225,19 @@ export class MdTooltip implements OnDestroy {
   }
 
   /**
-   * Dispose the tooltip when destroyed.
+   * Create the tooltip after init. The tooltip should be in the DOM ready to be displayed so that
+   * screen readers immediately have a reference to the tooltip content.
    */
-  ngOnDestroy() {
-    if (this._tooltipInstance) {
-      this._disposeTooltip();
+  ngOnInit() {
+    // Only create the tooltip if we are on the browser platform
+    if (this._platform.isBrowser) {
+      this._createTooltip();
```

Comment:
Creating the tooltip during init might end up causing a changed-after-checked error because you're attaching a view in the middle of change detection.